atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/AbstractDbWriteTable.java

```diff
@@ -135,4 +138,39 @@ public void delete(List<Entry<Cell, Long>> entries) {
                 + "  AND m.ts = ?",
                 args);
     }
+
+    @Override
+    public void delete(RangeRequest range) {
+        String prefixedTableName = prefixedTableNames.get(tableRef);
+        StringBuilder query = new StringBuilder();
+        query.append(" /* DELETE_RANGE (").append(prefixedTableName).append(") */ ");
```

Comment:
Huh, we should probably be using string builders everywhere for our sql construction... gulp